ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -188,8 +188,8 @@ tests:
       TEST_TIMEOUT: "30"
     test:
     - ref: cucushift-helper-service-build-info
-    - ref: idp-htpasswd
```

Comment:
@LiangquanLi930 please keep it. it is reserved for late usage. some golang cases will use these users.

Reply:
`idp-htpasswd` not support hyperhshift guest cluster

Reply:
https://github.com/openshift/release/pull/33365

Reply:
I do think #33365 has problem. will talk to you.

Reply:
ok, got it. thanks

Reply:
https://github.com/openshift/release/pull/33365 merged